src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -139,24 +139,24 @@ public class LibraryServiceSettings extends ServiceApiSettings {
 
   private final SimpleCallSettings<CreateShelfRequest, Shelf> createShelfSettings;
   private final SimpleCallSettings<GetShelfRequest, Shelf> getShelfSettings;
-  private final PageStreamingCallSettings<ListShelvesRequest, ListShelvesResponse, ListShelvesPagedResponse> listShelvesSettings;
+  private final PageStreamingCallSettings<ListShelvesRequest, ListShelvesResponse, PagedResponseWrappers.ListShelvesPagedResponse> listShelvesSettings;
```

Comment:
Maybe we could do a static import so that this doesn't get unwieldy? It might require some fancy typing work.